core/sql/sqlcomp/PrivMgrMD.h

```diff
@@ -110,12 +106,35 @@ typedef struct {
 
 } ViewUsage;
 
-typedef struct {
+// -----------------------------------------------------------------------
+// Class definitions
+// -----------------------------------------------------------------------
+class ObjectReference
+{
+  public:
+
+  ObjectReference()
+  : objectUID(0),
+    objectOwner(NA_UserIdDefault),
+    objectType(COM_UNKNOWN_OBJECT),
+    columnReferences(NULL),
+    updatedPrivs()
+  {}
+    
+  virtual ~ObjectReference ( void )
+  {
+    while(!columnReferences->empty())
+      delete columnReferences->back(), columnReferences->pop_back();
```

Comment:
Maybe I'm missing something. Isn't "delete columnReferences->back(), columnReferences->pop_back()" equivalent to { delete columnReferences->back(); columnReferences->pop_back(); }?